scripts/deploy.sh

```diff
@@ -1,9 +1,17 @@
+#!/bin/sh
```

Comment:
For consistency, should use a bash shebang to match `scripts/email.sh`
```suggestion
#!/bin/bash
```